src/components/QuizGameQuestions/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import "./index.css";
+
+// this function will display the fetched data to the dom.those parameters will pass in quizgame component.
+// the callback is used to check if the answer is true or not.
+// types of props = {
+//  question: string
+//  answerChoices: string
+//  handleClick: function
+//  userAnswer: string
+//  questionNum: number
+//  totalQuestions: number
+//}
+
+const QuestionCard = ({
+  question,
+  answerChoices,
+  handleClick,
+  userAnswer,
+  questionNum,
+  totalQuestions,
+}) => (
+  <div className="quizQuestionWrapper">
+    <p className="quizQuestionWrapper__numOfQuizQuestion">
+      Question: {questionNum} / {totalQuestions}
+    </p>
+    <p dangerouslySetInnerHTML={{ __html: question }} />
+    <div>
+      {answerChoices.map((answer) => (
+        <div key={answer}>
+          <button
+            className="quizQuestionWrapper__userAnswer"
+            value={answer}
```

Comment:
it still uses answer and not userAnswer

Reply:
i removed every thing related to userAnswer, now i have only answer i will fix also in the other components

Reply:
never mind i made a mistake reviewing here, i see answer is coming from line 27